app.test.js

```diff
@@ -90,5 +90,31 @@ describe("/api/articles/:article_id", () => {
 })
 
 describe("/api/articles", () => {
-
+  test("returns an array of article objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body}) => {
+        const articles = body.articles
+        articles.forEach((article) => {
```

Comment:
when using a forEach in a test, you should always check that the length of the array (articles) is higher than 0 (or a specific number if you know it!) first :)